spec/features/wiki/crud_spec.rb

```diff
@@ -13,9 +13,9 @@
     click_button "Log in"
 
     # localhost/wikis/new  wikis#index
-    expect(page).to have_content("Hello test@email.com")
+    expect(page).to have_content("Hey test@email.com")
     expect(current_path).to eq(wikis_path)
-    click_link "New wiki"
+    click_link "Public wiki" || "Private wiki"
```

Comment:
Ah ok, to be honest when I think about it, there will ALWAYS be the ability to create a public wiki so just sticking with that would suffice, but for future ref is there a nice way of implementing this so that one spec covered off both potential outcomes?